src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const arrDate = date.split(`${fromFormat[3]}`);
+  let str = '';
```

Comment:
don't use short forms for variable names
+ name should explain what is it

Reply:
Hi, I see you haven't fixed previous comments, maybe you did not push your changes?
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂